services/system/AccountSys/ui/src/appInit.ts

```diff
@@ -59,69 +207,78 @@ export const initAppFn = (setAppInitialized: () => void) =>
                     );
                 },
             },
             {
-                id: "getAuthedTransaction",
-                exec: async ({ transaction }: execArgs): Promise<any> => {
-                    const [user, accounts] = await Promise.all([
-                        query<null, string>(
-                            accountSysApplet,
-                            "getLoggedInUser"
-                        ),
-                        getJson<{ accountNum: string; authService: string }[]>(
-                            "/accounts"
-                        ),
-                    ]);
-
-                    const sendingAccount = accounts.find(
-                        (account) => account.accountNum === user
-                    );
-                    if (!sendingAccount) {
-                        console.error("No sending account", sendingAccount, {
-                            loggedInUser: user,
-                            sending: transaction.actions[0].sender,
-                        });
-                        throw new Error("No sending account found");
-                    }
-                    const isSecureAccount =
-                        sendingAccount.authService === "auth-ec-sys";
-                    if (isSecureAccount) {
-                        const keys = JSON.parse(
-                            localStorage.getItem("keyPairs") || "[]"
-                        ) as KeyPairWithAccounts[];
-
-                        const foundKey = keys.find((key) => {
-                            if (key.knownAccounts) {
-                                return key.knownAccounts.includes(user);
-                            }
-                        });
-
-                        if (foundKey) {
-                            const signingPrivateKey = foundKey.privateKey;
-                            return signTransaction("", transaction, [
-                                signingPrivateKey,
-                            ]);
-                        } else {
-                            console.error(
-                                `Failed to find the key pair for sender ${user}`
-                            );
-                            throw new Error(
-                                `Failed to find the key pair for sender ${user}`
-                            );
-                        }
-                    } else {
-                        return signTransaction("", transaction);
-                    }
+                id: "getKeyStoreAccounts",
+                exec: () => {
+                    return keystore.getAccounts();
+                },
+            },
+            {
+                id: "getProof",
+                exec: async (params: GetProofParams): Promise<any> => {
+                    console.info("getproof received!!", params);
+                    const proof = keystore.getProof(params);
+                    console.info("generated proof", proof);
+                    return { proof };
```

Comment:
Just a reminder to remove any console statements if they're no longer necessary.